package.json

```diff
@@ -31,11 +31,9 @@
   "keywords": [],
   "author": "",
   "license": "Apache-2.0",
-  "dependencies": {
-    "@floating-ui/dom": "^0.5.0"
```

Comment:
Since we are not building `core`, this dependency won't end up in the client's package lock unless specified here or in `peerDependencies`

Reply:
Updated in this commit: [157a05f](https://github.com/warp-ds/core/pull/15/commits/157a05f70416376c62d76fee4204ce06a72c01b5)